apps/prairielearn/src/pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs

```diff
@@ -221,7 +226,7 @@
               </th>
             </tr>
             <% } %>
-            <tr class="<% if (instance_question.sequence_locked) { %>bg-light pl-sequence-locked<% } %>">
+            <tr class="<% if (instance_question.sequence_locked || (instance_question.can_view !== null && !instance_question.can_view)) { %>bg-light pl-sequence-locked<% } %>">
```

Comment:
Generally prefer `!=` with `null` so that it will also catch `undefined`:
```suggestion
            <tr class="<% if (instance_question.sequence_locked || (instance_question.can_view != null && !instance_question.can_view)) { %>bg-light pl-sequence-locked<% } %>">
```